mdx/lib/mdx.ts

```diff
@@ -15,14 +15,16 @@ export async function emitMdx(filePath: string, pkg: PackageData) {
 ---
 ---
 
-import ReferenceWrapper from "@theme/ReferenceWrapper";
 import Description from "@theme/Description";
 import Definition from "@theme/Definition";
 import DefinitionHeader from "@theme/DefinitionHeader";
 import RpcDefinition from "@theme/RpcDefinition";
 import RpcDefinitionHeader from "@theme/RpcDefinitionHeader";
 import RpcDefinitionDescription from "@theme/RpcDefinitionDescription";
 import RpcMethodText from "@theme/RpcMethodText";
+import ProtosaurusImage from "@theme/ProtosaurusImage";
```

Comment:
Sorry if this question comes late: so, do we already have a plan on how users will have this "extension"? I mean, when they init a docusaurus project what's the next step? (if done manually).

Seems like through plugin mechanism, so the plugin can come as a separate npm module?

Reply:
Ah, yes, that's a very good point. I haven't thought of that at the moment. But yeah, I think theme plugin is the way to do it. For example: https://github.com/facebook/docusaurus/tree/main/packages/docusaurus-theme-classic

Then _maybe_, in users' `docusaurus.config.js`, they do this:

```
module.exports = {
  ...
+ themes: ['@protosaurus/theme-protosaurus'],
  ...
}
```

The same can be done for the `proto-messages` plugin, which users will add to the `rehypePlugins`.

In that case, perhaps we can have a `packages` folder, containing `theme-protosaurus` and `rehype-protosaurus`, where both plugins will be published to npm? We can think about using stuff such as Lerna after that.

Reply:
SGTM. Thank you!